src/components/course-calendar/plan-details.cjsx

```diff
@@ -37,6 +37,9 @@ CoursePlanDetails = React.createClass
     editLinkName = camelCase("edit-#{type}")
 
     reviewButton = @renderReviewButton()
+    editButton = <Router.Link to={editLinkName} params={linkParams}>
+      <BS.Button>Edit Assignment</BS.Button>
+    </Router.Link> if plan.isEditable
 
```

Comment:
Hmmm, this something I do semi-consistently throughout other code I've written.  The other way I've written it is:
```coffee
    if plan.isEditable
      editButton = <Router.Link to={editLinkName} params={linkParams}>
          <BS.Button>Edit Assignment</BS.Button>
        </Router.Link>
```